js/foam/u2/UpdateView.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CLASS({
+  package: 'foam.u2',
+  name: 'UpdateView',
+  extends: 'foam.u2.View',
```

Comment:
The plan for the replacement of `UpdateDetailView` is actually to default to auto-saving for now. Even if there are save/cancel actions, volunteering them to a toolbar is a separate concern. So I agree that the replacement for `UDV` need not be a `View`.

I looked at `ObjectReplicator` and it seems like it should work. I think the plan is as follows: use `ObjectReplicator` or a new, similar model to wrap the data and auto-save it. Then we construct a `DetailView` normally and pass it the replicator as its `data`. We might need a U2 `FutureView`  for that to work nicely.

Sounds like something for the next weekly design meeting.